lang/java/reef-webserver/src/main/java/org/apache/reef/webserver/DefaultHttpServerImpl.java

```diff
@@ -16,18 +16,33 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.reef.runtime.yarn.driver;
 
-import javax.inject.Inject;
+package org.apache.reef.webserver;
 
-final class DefaultTrackingURLProvider implements TrackingURLProvider {
+import javax.inject.Inject;
 
+/**
+ * Empty implementation for the HttpServer.
+ */
+public final class DefaultHttpServerImpl implements HttpServer {
```

Comment:
I would rather call the class `NoOpHttpServerImpl` or something like it, to emphasize that it has no functionality and serves only as an injectable placeholder.